modulo5/LabECommerce/src/endpoints/RegisterPurchases.ts

```diff
@@ -0,0 +1,69 @@
+import { Request, Response } from "express";
+import connection from "../database/connection";
+import { Purchases } from "../types/types";
+
+
+
+export const RegisterPurchases = async (req: Request, res: Response): Promise<void> => {
+    try {
+        let statusCode = 400
+        const { user_id, product_id, quantity } = req.body;
+
+        const checkUser = await connection.raw(`
```

Comment:
Já que você só usa essa o valor na posição 0 desse array, poderia desestrutura-lo aqui na definição da variavel. 
```suggestion
        const [checkUser] = await connection.raw(`
```
e então na linha 32 poderia colocar apenas
```typescript
checkUser.length > 0
```